diagnostic.md

```diff
@@ -76,8 +77,7 @@ var groceryList = [
 
 What is the value of `groceryList[2]`?
 
-_Replace this text with your answer._
-
+Nothing, maybe
```

Comment:
The prompt is asking for the element in 'groceryList' at index 2. 'groceryList' is an array (you can tell by the square brackets []. In arrays, all indexes start with 0, which means we are looking for the 3rd element in the groceryList. The answer should be 'a stick of butter'.